coding/javascript.md

```diff
@@ -155,7 +156,7 @@ const Car = ({ colour, ...wheelProps }: Props): JSX.Element => (
 // In use
 <Car colour="red" spinners={false} />
 
-renders => "<p>My car is red. My car is boring.</p>"
+// renders => "<p>My car is red. My car is boring.</p>"
```

Comment:
Why comment this?

Reply:
The rest of the code in this snippit was real code....this line is not, so I commented it out as it was likely meant as one originally.